pkg/doc/bbs/bbs12381g2pub/internal/bls12_381.go

```diff
@@ -0,0 +1,26 @@
+/*
+Taken from https://github.com/kilic/bls12-381
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package internal
```

Comment:
you should make a subfolder under this internal folder, so the package still has a descriptive name.

Reply:
done - added `kilic/bls12-381`.